inst/shiny/modules/prep_mesh.md

```
@@ -21,52 +21,54 @@ are adjusted depending on the response data and so the examples are shown relati
 - The *Max edge* parameter controls the maximum distance between nodes and has two values for controlling the parameter inside and outside of the area of interest respectively. This parameter has the 
 largest effect on the number of nodes: 
 
+
 <p align="middle">
-  <img src="figures/mesh_default.png" height = 400 />
-  <img src="figures/mesh_max_edge_div_2.png" height = 400 /> 
-  <img src="figures/mesh_max_edge_div_4.png" height = 400 />
+  <img src="https://raw.githubusercontent.com/simon-smart88/disagapp/44495a7e5e57d12e1a93bd9f9b0105351e588203/inst/shiny/www/figures/mesh_default.png" height = 400 />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)